plex/config.yaml

```diff
@@ -18,6 +18,9 @@ map:
   - ssl
 devices:
   - /dev/dri
+  - /dev/dri/card0
+  - /dev/dri/card1
+  - /dev/dri/renderD128
```

Comment:
There are a duplicate of the above already.

Please explain how these behave differently.

Note: I do run/test this add-on on Intel Alderlake and hardware acceleration works fine on that.

Reply:
I am not sure why, but with the Alder Lake and Raptor Lake generation it did show the GPU inside PLEX, but it never triggered Hardware acceleration, just did the transcode in software. Auto or manually selecting the GPU that was detected did not change it.

When I add these three mappings (not sure which of the three is actually used), it works as expected.

Reply:
ok, I also see no harm in adding them. Let's do it.